Sources/StreamChatUI/Components.swift

```diff
@@ -327,6 +330,9 @@ public struct Components {
     /// A view that displays the video attachment preview in composer.
     public var videoAttachmentComposerPreview: VideoAttachmentComposerPreview
         .Type = VideoAttachmentComposerPreview.self
+    
+    /// A formatter used for text Markdown
+    public var markdownFormatter: MarkdownFormatter.Type = DefaultMarkdownFormatter.self
```

Comment:
In fact, the `markdownFormatter` and `markdownFormatterEnabled` should be in the `Appearance.formatters` 🙂

Reply:
Done 🙂